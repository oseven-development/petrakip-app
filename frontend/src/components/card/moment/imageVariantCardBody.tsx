import { IonImg } from '@ionic/react'

export interface ImageVariantProps {
  variant: 'image'
  imageSrc: string
}

export const ImageVariantCardBody: React.FC<ImageVariantProps> = ({
  imageSrc,
}) => <IonImg src={imageSrc} />
