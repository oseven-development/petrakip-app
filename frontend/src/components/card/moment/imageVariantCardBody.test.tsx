import { render } from '@testing-library/react'

import { ImageVariantCardBody } from './imageVariantCardBody'

describe('ImageVariantCard Component', () => {
  const audioVariantCard = (
    <ImageVariantCardBody variant="image" imageSrc="mySource" />
  )

  test('renders without crashing', () => {
    const { container } = render(audioVariantCard)
    expect(container).toMatchSnapshot()
  })
})
