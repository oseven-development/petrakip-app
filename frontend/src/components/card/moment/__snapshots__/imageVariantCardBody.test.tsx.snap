// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageVariantCard Component renders without crashing 1`] = `
<div>
  <ion-img
    src="mySource"
  />
</div>
`;
